### PR TITLE
.format replaced with f "" 

### PR DESCRIPTION
**Pull request by SergeLeon**
.format replaced with f "" wherever possible, except for tests

### CODE REVIEW

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
please solve this merge conflict

File: mslib/msui/wms_control.py [hunk 1]
```diff
@@ -660,9 +665,11 @@ def get_capabilities(self):
         else:
             # url shortener url translated
             url = request.url
-            url = url.replace('?service=WMS&request=GetCapabilities&version=1.1.1', '')
+            
+            url = url.replace("?service=WMS", "").replace("&service=WMS", "") \	
+                     .replace("?request=GetCapabilities",
```
Comment (Marilyth) [inline mslib/msui/wms_control.py]: Something seems very awry here

**Review by Marilyth — DISMISSED** [review]

File: mslib/msui/wms_control.py (hunk 1 above)
Comment (SergeLeon) [reply mslib/msui/wms_control.py]: I solved all the problems of the merge

**Review by ReimarBauer — COMMENTED** [review]
What changed on that file ? mslib/mswms/xml_templates/get_capabilities130.pt

**Review by ReimarBauer — DISMISSED** [review]

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
That is allmost done. There are two files too much in this pull request

service_exception130.pt and service_exception130.pt seems to have changes in the amout of whitespace. 

We need to take care for the issue content and the work done in the PR.
This PR is finished when the service_exception130.pt and service_exception130.pt  are changed to origin state.

Later on a release we can lookup all closed issues related to the released version. If there are other changes done that is not easily to keep track.

**Comment by ReimarBauer** [comment]
@SergeLeon let me know if you want to join our slack channel. 
https://open-mss.github.io/develop/GSOC2021/ideas

**Comment by ReimarBauer** [comment]
> That is allmost done. There are two files too much in this pull request
> 
> service_exception130.pt and service_exception130.pt seems to have changes in the amout of whitespace.
> 
> We need to take care for the issue content and the work done in the PR.
> This PR is finished when the service_exception130.pt and service_exception130.pt are changed to origin state.
> 
> Later on a release we can lookup all closed issues related to the released version. If there are other changes done that is not easily to keep track.


OK, good work over all :)

**Review by ReimarBauer — APPROVED** [review]

**Pull request merged**